exercises/reverse-string/canonical-data.json

```diff
@@ -0,0 +1,39 @@
+{
+  "exercise": "reverse-string",
+  "version": "1.0.0",      
+  "comments": [
+        "If property based testing tools are available, a good property to test is reversing a string twice: reverse(reverse(string)) == string"
+      ],
+  "cases": [
+    {
+      "description": "empty string",
+      "property": "reverse",
+      "input": "",
+      "expected": ""
+    },
+    {
+      "description": "a word",
+      "property": "reverse",
+      "input": "robot",
+      "expected": "tobor"
+    },
+    {
+      "description": "a capitalized word",
+      "property": "reverse",
+      "input": "Ramen",
+      "expected": "nemaR"
+    },
+    {
+      "description": "a sentence with punctuation",
+      "property": "reverse",  
+      "input": "I'm hungry!",
+      "expected": "!yrgnuh m'I"
+    },
+    {
+      "description": "a string that reads the same backward as forward (i.e. palindrome)",
```

Comment:
Continuing discussion from: https://github.com/exercism/problem-specifications/pull/960#discussion_r149814157

It is helpful for descriptions to be short and to the point, so either 

"a string that reads the same backwards and forwards"
or
"a palindrome"

but not both.

My preference is for "a palindrome", because that is the briefest description of what it is.
It's still possible to solve the problem without knowing the definition of the word and the definition can easily be found by a search if necessary.

Reply:
To give a concrete example of why concision is useful, consider the output of programs which generate tests based on the canonical data. Which is best:

- `fn test_a_palindrome() { ... }`
- `fn test_a_string_that_reads_the_same_backwards_and_forwards() { ... }`
- `fn test_a_string_that_reads_the_same_backward_as_forward_ie_palindrome() { ... }`

Depending on which option gets merged, the Rust test generator will create one of the above. IMO, the shortest is the best.

Reply:
Ok I'm convinced. I added just "a palindrome" for description to latest commit.

Please review @Insti. Thanks.